pkg/types/types.go

```diff
@@ -195,5 +195,5 @@ func prettyPrint(obj interface{}) error {
 	return err
 }
 
-// NotImplementedError is used to indicate that a method is not implemented for the given platform
-var NotImplementedError = errors.New("Not Implemented")
```

Comment:
That is desired but it has quite huge impact on API.

BTW. I don't see any usage of this variable anywhere in this repo and in plugins repo. Maybe instead of renaming it - we should remove that at all?

Reply:
If we don't use it anywhere in CNI, then we should remove it. I'm not sure we ever used it anywhere either?

Reply:
For now, it seems that this has not ever been used in CNI repos. I'll remove this, thanks for your advice @jellonek  @dcbw

Reply:
Updated!